eth/filters/filter_system.go

```diff
@@ -641,9 +610,7 @@ func (es *EventSystem) lightFilterLogs(header *types.Header, addresses []common.
 		// Get the logs of the block
 		ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
 		defer cancel()
-		indexed := es.sys.backend.LastBloomIndex()
```

Comment:
Is `LastBloomIndex()` still necessary or can we revert this change from the prior PR?

Reply:
We could revert. I considered doing this as well.

Reply:
Removed